### PR TITLE
DAMD-32: Split pfsConfig into pre- and post- mapping files

### PR DESCRIPTION
**Pull request by PaulPrice**

### CODE REVIEW

File: datamodel.txt [hunk 1]
```diff
@@ -214,16 +214,51 @@ lists fiberId, catId, objId, ra, dec, fiber flux, MCS centroid, ... for each obj
       ra             32-bit float
       dec            32-bit float
```
Comment (PaulPrice) [inline datamodel.txt]: Need 64 bits for sub-arcsecond position.

File: datamodel.txt [hunk 2]
```diff
@@ -214,16 +214,51 @@ lists fiberId, catId, objId, ra, dec, fiber flux, MCS centroid, ... for each obj
...
       fiberMag       5*32-bit float
```
[the remark this answers is not in the document]
Comment (CraigLoomis) [reply datamodel.txt]: There was some discussion on this on the datamodel slack. For the survey we will know what the photo mags mean, but not necessarily for open use. 

The photo mags are used both by the DRP and by the observing QA.

Arguably having 5 slots and 5 names will be flexible enough for any targeting. We wondered whether we/NOAJ wants to limit that to known bandpasses, or require some synthetic bandpasses, etc.

File: datamodel.txt [hunk 3]
```diff
@@ -214,16 +214,51 @@ lists fiberId, catId, objId, ra, dec, fiber flux, MCS centroid, ... for each obj
...
+
+Another is that we will know when the cobras have been moved, and we
+can record the first PFS visitId for which the PFI has been configured
+according to the `pfiDesign`. This is referred to as the pfiConfig's
+`visit0`. From this, the filename is:
+
+    "pfiConfig-%06d-0x%08x.fits" (visit0, pfiDesignId)
+
+Finally, we are likely to have multiple `pfiConfig` files with the same
+`pfiConfigId` but different `visit0`. That will indicate that the field
```
[the remark this answers is not in the document]
Comment (CraigLoomis) [reply datamodel.txt]: Same design, different observing runs. So one pfiDesign, multiple pfiConfigs

File: datamodel.txt [hunk 4]
```diff
@@ -214,16 +214,51 @@ lists fiberId, catId, objId, ra, dec, fiber flux, MCS centroid, ... for each obj
...
+`pfiDesign` FITS cards and the DESIGN HDU.
+
+We add a second HDU, containing as much MCS and PFI info as we find
+useful. Starting with:
+
+HDU #2 CONFIG
+  fiberId     32-bit int, matching one line in the pfiDesign file
+  pfiX, pfiY  32-bit floats, giving the final position in PFI mm.
+  mcsX, mcsY  32-bit floats, giving the final position in MCS pixels.
+  fpsTransformId 64-bit int, indicating the version of the transforms.
```
[the remark this answers is not in the document]
Comment (CraigLoomis) [reply datamodel.txt]: Yeah. We don't know the details yet. But we expect to have a model function for the MCS-to-PFI distortion, plus some varying coefficients (in, respectively, the `pfs_utils` and `pfs_instdata` products)

So the git SHAs for the coefficients, at least.

File: datamodel.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (PaulPrice) [reply datamodel.txt]: I was thinking to allow any bandpass that we have a transmission curve for. Open use users could use different bandpasses, so long as they provide the corresponding transmission curve.

[the remark this answers is not in the document]
Comment (PaulPrice) [reply datamodel.txt]: It's not just the targeting that I'm concerned about, but also the flux calibration. grizy is sufficient to calibrate the blue and red arms, but maybe not do as well on the NIR arm.

File: datamodel.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (PaulPrice) [reply datamodel.txt]: Right, so you do mean `pfiDesignId` here.

File: datamodel.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (CraigLoomis) [reply datamodel.txt]: That is what we were converging on. So all good.

File: datamodel.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (PaulPrice) [reply datamodel.txt]: And then that transformation will be written to the data repo using the same `fpsTransformId` so we can retrieve it by the same? Makes sense.

File: datamodel.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (CraigLoomis) [reply datamodel.txt]: Yes, sorry, I misread.

File: datamodel.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (CraigLoomis) [reply datamodel.txt]: OK, good.

**Comment by PaulPrice** [comment]
Closing this PR in favour of a new implementation.

**Pull request closed without merging**